src/reactions/edit.js

```diff
@@ -65,13 +150,45 @@ export default function Edit( { __unstableLayoutClassNames } ) {
 
 	return (
 		<div { ...blockProps }>
+			{ isSummaryStyle && (
+				<InspectorControls>
+					<PanelBody title={ __( 'Summary Settings', 'activitypub' ) }>
+						<ToggleControl
+							label={ __( 'Show Comments', 'activitypub' ) }
+							help={ __( 'Include the comment count in the summary.', 'activitypub' ) }
+							checked={ showComments }
+							onChange={ ( value ) => setAttributes( { showComments: value } ) }
+							__nextHasNoMarginBottom
+						/>
+						<ToggleControl
+							label={ __( 'Show empty Reactions', 'activitypub' ) }
+							help={ __( 'Display Reaction types even when they have no count.', 'activitypub' ) }
+							checked={ showEmpty }
+							onChange={ ( value ) => setAttributes( { showEmpty: value } ) }
+							__nextHasNoMarginBottom
+						/>
```

Comment:
Same as above, let's remove reactions that don't have a count, in the dummy that would be quotes.